src/components/NcRichText/autolink.js

```diff
@@ -84,20 +84,37 @@ export const parseUrl = (text) => {
 	return text
 }
 
-export const getRoute = (router, url) => {
+/**
+ * Try to get mapped path from router's routes
+ *
+ * @param {import('vue-router').default} router - VueRouter instance
+ * @param {string} urlString - URL to parse
+ * @return {string|null} path (RouteLocationPathRaw) for router link or null if path does not match any route
+ */
+export const getRoute = (router, urlString) => {
 	// Skip if Router is not defined in app, or baseUrl does not match
-	if (!router || !url.includes(getBaseUrl())) {
+	if (!router || !urlString.startsWith(getBaseUrl())) {
```

Comment:
`next` branch is diverged, with checking for base/root URL depending whether absolute url was provided
Shall we align with that changes?